tests/unit/Hooks/UserHooksTest.php

```diff
@@ -396,7 +396,11 @@ protected function setUp(): void {
 		$this->recoveryMock = $recoveryMock;
 		$this->utilMock = $this->createMock(Util::class);
 		$this->utilMock->expects($this->any())->method('isMasterKeyEnabled')->willReturn(false);
+		$this->eventDispatcher = \OC::$server->getEventDispatcher();
```

Comment:
I'd rather use a mock instance instead. I don't know if it will break things or not.